src/container/Layout/layout.js

```diff
@@ -30,7 +30,7 @@ class Layout extends Component {
 
     render() {
         return (
-            <div className="container-fluid p-0">
+            <div className="container-fluid">
                 <Switch>
                     <Route path="/" exact  component={Home}/>
                     <Route path="/signup" exact component={Signup}/>
```

Comment:
signup or SignUp or Signup??????? Choose one convention. Look for the other websites how they do it

Reply:
fixed